src/types.ts

```diff
@@ -146,6 +148,7 @@ export interface MarketplaceVizHelpers {
   dependToggle: (section: string, label: string, init: boolean, parentKey?: string, parentObj?: any) => VisOption,
   dependString: (section: string, label: string, init: string, parentKey?: string, parentObj?: any) => VisOption,
   makeNumber: (section: string, label: string, init: number, order?: number, parent?: string, parentObj?: any) => VisOption
+  makeList: (section: string, label: string, init: string, choices: [], order?: number, parent?: string, parentObj?: any) => VisOption,
```

Comment:
The only thing I'm gonna change is adding a type to choices.
SelectOption[]
interface SelectOption {
  value: string
  label: string
  }